### PR TITLE
Task 96-120 : Dataset 3 - Phase 2

### PR DESCRIPTION
**Pull request by PruthaGaherwar**

### CODE REVIEW

**Comment by Ravsehajsinghpuri** [comment]
Hi @PruthaGaherwar No variations have been made of the different tasks, can you do that update the PR?

**Comment by PruthaGaherwar** [comment]
@Ravsehajsinghpuri Could you please elaborate on what exactly do you need me to do?

**Comment by Ravsehajsinghpuri** [comment]
Did you generate different variations of the tasks? I can see only 25 tasks which are the original tasks i suppose.

**Comment by PruthaGaherwar** [comment]
I generated one variation per task and these tasks here are the variation_1 of the original tasks

**Comment by Ravsehajsinghpuri** [comment]
@PruthaGaherwar 1) Can you add the original tasks in this PR. 2) Can you rename the task variants as taskxxx.1_yyyy, taskxxx.2_yyyy, taskxxx.3_yyyy and so on. For e.g. task547.1_spl_translation_entk_en.json, task547.2_spl_translation_entk_en.json, task547.3_spl_translation_entk_en.json and so on. Do this for all the tasks in this PR 3) Can you resolve the conflicts 4) Can you add 5-6 variants of each task?

**Comment by PruthaGaherwar** [comment]
@swarooprm As far as I recall, you had asked to create at least one to three variants of each task, right? I had created one variant which was supposed to be an acceptable submission. Could you please review this new change mentioned above and let me know on the further steps to be taken. Thanks

File: src/test_all.py [hunk 1]
```diff
@@ -15,7 +15,6 @@
     'Categories',
     'Source'
 ]
-
```
Comment (Ravsehajsinghpuri) [inline src/test_all.py:18]: Can you remove this change?

**Comment by Ravsehajsinghpuri** [comment]
> @swarooprm As far as I recall, you had asked to create at least one to three variants of each task, right? I had created one variant which was supposed to be an acceptable submission. Could you please review this new change mentioned above and let me know on the further steps to be taken. Thanks

The files which you have added are the variants? Or the original tasks? If they are the variants, then rename in the format I have specified and add the original tasks along with it

**Comment by Ravsehajsinghpuri** [comment]
> > @swarooprm As far as I recall, you had asked to create at least one to three variants of each task, right? I had created one variant which was supposed to be an acceptable submission. Could you please review this new change mentioned above and let me know on the further steps to be taken. Thanks
> 
> The files which you have added are the variants? Or the original tasks? If they are the variants, then rename in the format I have specified and add the original tasks along with it

friendly reminder on this -  try to make the changes by today or latest by tomorrow, thanks!

**Comment by PruthaGaherwar** [comment]
Hi @Ravsehajsinghpuri I have made the changes to the task files. can you please check the change made to the test_all.py file and let me know if that is what you wanted? if not I will change it again. Thanks.

**Comment by Ravsehajsinghpuri** [comment]
> Hi @Ravsehajsinghpuri I have made the changes to the task files. can you please check the change made to the test_all.py file and let me know if that is what you wanted? if not I will change it again. Thanks.

Have you added only 1 variant per task?

**Comment by PruthaGaherwar** [comment]
yes

**Pull request merged**